packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.spec.tsx

```diff
@@ -94,17 +95,17 @@ describe(disableAdd, () => {
             x: 0,
             y: 0,
             z: 0,
-            properties: { queryConfig: { query: { assets: [] } } },
+            properties: {},
             height: 500,
             width: 800,
           },
         ],
         10
       )
-    ).toBeFalsy();
+    ).toBeTruthy();
   });
 
-  it('should return false if x-y-plot is selected and has assets already added', () => {
+  it('should return false if x-y-plot is selected and more than 1 property selected', () => {
```

Comment:
The test name says more than 1 property selected, yet the properties object is empty?

Reply:
for x-y the properties added doesnt matter